src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -120,18 +121,20 @@ public async Task<string> SubmitPledge(CreatePostRequest request)
 
             ValidateCreatePledgeCacheItem(cacheItem);
 
-            var pledgeDto = new PledgeDto
+            var apiRequest = new CreatePledgeRequest
```

Comment:
I might call this variable `createPledgeRequest` to reflect its type

Reply:
Renamed